Cargo.toml

```diff
@@ -5,7 +5,9 @@ authors = ["ibaryshnikov"]
 edition = "2021"
 
 [lib]
-crate-type = ["staticlib"]
+path = "rust-bits/lib.rs"
+crate-type = ["staticlib", "cdylib"]
```

Comment:
does `cdylib` work here? I was expecting it to fail to compile because Rust parts depend on Haskell parts (freeStablePtr etc)

Reply:
yep, it generates `target/debug/libiced_hs.so` that is copied here: https://github.com/ibaryshnikov/iced-hs/pull/3/files#diff-95d596561bc01292f2cebe1ecbabf44af34eaee24a244de568e6489814ffcb0bR120

Reply:
That's confusing. I'll take a closer look later, but if it works we can go with it